DOCCORE-54 Clarify that dump cmd does not support backing up RBAC metadata (#2769)
neo4j/docs-operations: modules/ROOT/pages/backup-restore/modes.adoc

...
----
====

> The `neo4j-admin database backup` command with the `--type=full` argument creates a xref:backup-restore/online-backup.adoc#backup-artifact[full backup artifact] file in the format of _<databasename><timestamp>.backup_, called  in the target location (`--to-path`).

Reviewer: ```suggestion
The `neo4j-admin database backup` command with the `--type=full` argument creates a xref:backup-restore/online-backup.adoc#backup-artifact[full backup artifact] file in the format of _<databasename><timestamp>.backup_, in the target location specified by `--to-path`.
```

> The `neo4j-admin database backup` command with the `--type=full` argument creates a xref:backup-restore/online-backup.adoc#backup-artifact[full backup artifact] file in the format of _<databasename><timestamp>.backup_, called  in the target location (`--to-path`).

Author: Where do we mention the `neo4j-admin database backup full` command? Do we have it in the code?

The `neo4j-admin database backup` command allows you to include metadata, enabling the backup to store the role-based access control (RBAC) metadata associated with the database.

For more information about online backup options and how to control memory usage, see xref:backup-restore/online-backup.adoc[Back up an online database].

=== `neo4j-admin database dump`

> The `neo4j-admin database dump` command creates a single-file archive, that follows the format _<databasename><timestamp>.dump_, and stores it in the xref:configuration/file-locations.adoc#data[_<NEO4J_HOME>/data_] directory.

Reviewer: ```suggestion
The `neo4j-admin database dump` command creates a single-file archive, that follows the format _<databasename><timestamp>.dump_, and stores it in the xref:configuration/file-locations.adoc#data[_<NEO4J_HOME>/data_] directory unless specified by `--to-path=<path>`.
```

> Note that when using `neo4j-admin database dump`, only the database contents are backed up; database metadata and associated RBAC settings are not included.

Reviewer: What are these RBAC settings? Also, I think we need to find all places where we mention `--include-metadata` and update them accordingly, as by default the backup command backs up all users and roles, unless you specified something different from the default `all`.

```suggestion
Note that the dump archive contains only the database without the associated users and roles metadata.
```

Author: I think this is in line with our RBAC -- https://neo4j.com/docs/operations-manual/current/authentication-authorization/manage-privileges/

For more information about performing a full backup against an *offline* database, see xref:backup-restore/offline-backup.adoc[Back up an offline database].

[role=enterprise-edition]
[[differential-backup]]
== Differential backup

> In the online backup version, after the initial full backup, the subsequent backups attempt to use the differential mode, where just the delta of the transaction logs since the last backup are transferred and used to create a differential backup artifact (stored in the target location).

Reviewer: ```suggestion
In the online backup version, after the initial full backup, subsequent backups attempt to use differential mode, where only the delta of the transaction logs since the last backup is transferred and used to create a differential backup artifact (stored in the target location).
```
